docker/softlinkstatic.py

```diff
@@ -7,10 +7,11 @@
 with open(STATIC) as static_fp:
     static_files = json.load(static_fp)
 
-for path in static_files['paths']:
-    full_path = os.path.join('./static', path)
-
-    try:
-        os.unlink(full_path)
-    except OSError:
-        pass
+for orig, hashed in static_files['paths'].items():
+    if '?' in orig:
+        continue
```

Comment:
Oh cool. Of course you didn't, because there isn't one. Didn't realize that. Nice catch.